docs/usage.md

```diff
@@ -30,7 +30,31 @@ Add the `add_thumbnails` dependency to generate thumbnails for JPEG and PNG file
 c2pa = { version = "0.45.2", features = ["file_io", "add_thumbnails"] }
 ```
 
-NOTE: If you are building for WASM, omit the `file_io` dependency.
+### Building wasm
+
+Building wasm requires LLVM clang due to the `ring` crate. The version of clang that comes with XCode does not support wasm targets.
+
+1. On macOS, install clang with homebrew `brew install llvm`.
```

Comment:
Can you add the minimum llvm version to install (you can pick the one you currently have). So we are sure we know at least one working setup for sure.

Reply:
It's difficult to pin down a version, there has been wasm support in LLVM for a very long time. WASI is continuing to evolve, so newer LLVM versions will have better support.

@crandmck I don't understand the spirit of the question. There are many supported LLVM targets for this crate and we don't have any language indicating why you would compile for them. The user will necessarily have a use case outside the scope of this crate and know which target to compile for. No one has to compile for any target if they don't want to.

Reply:
IIUC, one can use c2pa-rs directly (i.e. not for another language binding).  For example, for c2patool.  Maybe I'm misunderstanding.

Reply:
Note that make test has always tried to build wasm, and that will no longer work on macs without installing llvm. We may want to reconfigure this and make wasm/wasi builds as separate step.